src/mixins/namex-api-mixin.ts

```diff
@@ -43,7 +43,7 @@ export default class NameXApiMixin extends Vue {
   async queryNameRequest (nrNumber: string): Promise<any> {
     if (nrNumber) {
       // TODO FIX - Override until we can proxy through legal api
-      let url = `https://namex-dev.pathfinder.gov.bc.ca/api/v1/requests/NR ${nrNumber}`
+      let url = `https://namex-dev.pathfinder.gov.bc.ca/api/v1/requests/${nrNumber}`
```

Comment:
Needs to be a property.

Reply:
Sorry, this was code already in. You'd like to see this in the config map?

Reply:
@severinbeauvais @cameron-freshworks this call is meant to be proxied through legal api when that endpoint is ready, do we need a property for it?

Reply:
Need clarification here :)

Reply:
I forgot this will eventually be proxied and changed. Leave this for now.

Reply:
Wow, I even missed Ody's comments. Sorry!